pages/discover.js

```diff
@@ -93,10 +92,10 @@ export default function Discover() {
             Managing the data where it is created.
           </h3>
           <div className="pt-4">
-            CDS avoids moving data to centralized infrastructures by computing it directly where it
-            is created, reducing this way the data transmission efforts. CDS can select the piece of
-            data to share with indexing servers, limiting it, for example, to data intended to
-            produce analyses or commercial services.
+            CDS avoids the use of centralized infrastructures by computing data directly where it is
```

Comment:
the execution of data in centralized infra... The point is not to process personal data in a delocalized place but directly on the device.